docs/admin/authentication.md

```diff
@@ -486,6 +486,21 @@ An unsuccessful request would return:
 
 HTTP status codes can be used to supply additional error context.
 
+#### Using OpenStack Keystone authentication Webhook
+
+There are two parts to authentication with OpenStack Keystone. On the client-side, you can use `--auth-provider=openstack` option to reuse the `OS_*` environment variables:
+```
+kubectl config set-credentials openstackuser --auth-provider=openstack
+```
+Note that you will also need to use `kubectl config set-context` and `kubectl config use-context` to activate the context. Please make sure you have `OS_DOMAIN_NAME` to use Keystone V3 API.
+
+On the server-side, one choice is to use the webhook from:   
+https://github.com/dims/openstack-cloud-controller-manager/blob/master/cmd/k8s-keystone-auth/main.go
+
+Documentation for the webhook is at  
+https://github.com/dims/openstack-cloud-controller-manager/blob/master/docs/using-keystone-webhook-authenticator-and-authorizer.md
+
+This webhook from folks in the OpenStack community is just one option. Other webhooks may be available with alternative user mappings in the future. 
```

Comment:
Still not available for now. Shall we remove this line?

Reply:
Recommend dropping the line about "other webhooks may be available..." - better to focus on what is now and how to utilize it vs. what could come.